src/core/tests/SparseMatrixUnitTest.cpp

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright (C) 2015 Fondazione Istituto Italiano di Tecnologia
+ * Authors: Silvio Traversaro
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ *
+ */
+
+#include <iDynTree/Core/SparseMatrix.h>
+#include <iDynTree/Core/Triplets.h>
+#include <iDynTree/Core/TestUtils.h>
+#include <iostream>
+
+
+using namespace iDynTree;
+using namespace std;
+
+void testCreateMatrixFromAccessorOperator()
+{
+    std::cout << "------------------------------------" << std::endl;
+    std::cout << "testCreateMatrixFromAccessorOperator" << std::endl;
+    SparseMatrix matrix(5,5);
+    matrix(2,0) = 7;
+    std::cout << matrix.description() << std::endl;
+    matrix(0,1) = 3;
+    std::cout << matrix.description() << std::endl;
+    matrix(1,0) = 22;
+    std::cout << matrix.description() << std::endl;
+    matrix(1,4) = 17;
+    std::cout << matrix.description() << std::endl;
+    matrix(2,1) = 5;
+    std::cout << matrix.description() << std::endl;
+    matrix(2,3) = 1;
+    std::cout << matrix.description() << std::endl;
+    matrix(4,2) = 14;
+    std::cout << matrix.description() << std::endl;
+    matrix(4,4) = 8;
+    std::cout << matrix.description() << std::endl;
+
+    std::cout << "Matrix:\n" << matrix.description(true) << std::endl;
+    std::cout << "------------------------------------" << std::endl;
```

Comment:
This tests are a bit strange, as they have no assertions in it. There are intended in this way or a part is missing?